cmd/registration-dns/config.toml

```diff
@@ -0,0 +1,12 @@
+# Address on which to listen for dns registration requests
+addr = "[::]:53"
+
+# URLs for unidirectional and bidirectional registration API
+api_url = "http://localhost:8080/register"
+bdapi_url = "http://localhost:8080/register-bidirectional"
+
+# Base domain authoritive to nameserver
+domain = "r.refraction.network"
+
+# Path to private key file for dns registrar (generated with -genKey flag)
+private_key_path = "/opt/conjure/cmd/registration-dns/privkey"
```

Comment:
We should be using the stations public / private key pairs for the DNS registrar or some version that we can reliably generate from the station keys. Just so we don't have to add a separate key pair to distribution and configuration.

As long as the keys used here by the noise protocol are ed25519 then we should be totally fine since that is the conjure key system too. It might even be as simple as changing the paths in the configs on the station side.

This affects the changes in:
* `cmd/registration-dns/config.toml`
* `simulation/phantombox/config/registration-dns/config.toml`
* `simulation/phantombox/station/user-data`

Reply:
That make sense. Seems like the conjure keys are indeed compatible as the key pair from the simulation environment works just fine. I've changed the way to read the private key so that it works with the conjure key file format. Also changed the default file path for the private key file.